integrations/docker/images/chip-build-cirque/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION}
+
+# Bazel
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
+       curl gnupg \
+    && curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg \
+    && mv bazel.gpg /etc/apt/trusted.gpg.d/ \
+    && echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
+       bazel \
+    && : # aids diffs
+
+# Docker
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
+       curl gnupg-agent apt-transport-https ca-certificates \
+       software-properties-common \
+    && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - \
+    && add-apt-repository \
+       "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
+       docker-ce docker-ce-cli containerd.io \
+    && : # aids diffs
+
+# Other Cirque prereqs
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
+       sudo socat psmisc tigervnc-standalone-server tigervnc-viewer \
```

Comment:
as I know, we always running as root in docker, do we still need to install sudo in docker container?

Reply:
prolly not...

Reply:
can fix later, once sudos are removed from cirque scripts